docs/csharp/linq/perform-a-subquery-on-a-grouping-operation.md

```diff
@@ -17,7 +17,11 @@ For more information about continuations, see [into](../language-reference/keywo
 > [!NOTE]
 > This example contains references to objects that are defined in the sample code in [Query a collection of objects](query-a-collection-of-objects.md).
 
-[!code-csharp[csProgGuideLINQ#23](~/samples/snippets/csharp/concepts/linq/how-to-perform-a-subquery-on-a-grouping-operation_1.cs)]  
+[!code-csharp[csProgGuideLINQ#23](~/samples/snippets/csharp/concepts/linq/how-to-perform-a-subquery-on-a-grouping-operation_1.cs)] 
+
+The query in the snippet above can also be written using method syntax. The code snippet below has a semantically equivalent query written using method syntax.
```

Comment:
nit: The code snippet below --> The following code snippet

Reply:
Thanks for the suggestion! I like that phrasing better.